.pre-commit-config.yaml

```diff
@@ -38,8 +38,6 @@ repos:
           - types-PyYAML
           - types-setuptools
           - types-psutil
-          # Libraries exclusively imported under `if TYPE_CHECKING:`
-          - typing_extensions  # To be reviewed after dropping Python 3.7
```

Comment:
This is needed again after https://github.com/dask/distributed/pull/5814 as it uses ``TypeAlias``

Reply:
Would it make sense to add in that PR?

Reply:
yes no problem